starfish/test/full_pipelines/cli/_base_cli_test.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import unittest
+from typing import List, Tuple
+
+from starfish.intensity_table import IntensityTable
+from starfish.util import clock
+
+
+class CLITest(unittest.TestCase):
+    __test__ = False
+
+    SUBDIRS = Tuple[str]
+    STAGES = Tuple[List]
+
+    def verify_results(self, intensities):
+        pass
+
+    def test_run_pipline(self):
+        tempdir = tempfile.mkdtemp()
+        coverage_enabled = "STARFISH_COVERAGE" in os.environ
+        print(tempdir)
+
+        def callback(interval):
+            print(" ".join(stage[:2]), " ==> {} seconds".format(interval))
+
+        try:
+            for subdir in self.SUBDIRS:
```

Comment:
Can you make use of `starfish.util.exec` here?

Reply:
Didn't see that! yes totally, I'll refactor